src/main/kotlin/step2/Calculator.kt

```diff
@@ -0,0 +1,28 @@
+package step2
+
+object Calculator {
+
+    fun calculate(input: String): Double {
+
+        val inputList: List<String> = splitInput(input)
+
+        var result: Double = inputList[0].toDouble()
+
+        for (i: Int in 1 until inputList.size step 2) {
+            result = computeByOperator(inputList[i], result, inputList[i + 1].toDouble())
+        }
+
+        return result
+    }
+
+    @Throws(IllegalArgumentException::class)
+    fun splitInput(input: String): List<String> {
+        val inputList: List<String> = input.split(' ')
+        require(inputList.size % 2 == 1) { ErrorMessage.INCORRECT_INPUT }
+
+        return inputList
```

Comment:
```suggestion
        return input.split(' ').also { require(inputList.size % 2 == 1) { ErrorMessage.INCORRECT_INPUT } }
```
이렇게도 쓸 수 있어요! (역시나 반드시 이렇게 쓸 필요는 없습니다)
 [이 글](https://medium.com/@limgyumin/%EC%BD%94%ED%8B%80%EB%A6%B0-%EC%9D%98-apply-with-let-also-run-%EC%9D%80-%EC%96%B8%EC%A0%9C-%EC%82%AC%EC%9A%A9%ED%95%98%EB%8A%94%EA%B0%80-4a517292df29)을 읽어보고 코틀린의 여러 스코프 함수들에 대해서 학습해보시면 좋을 것 같습니다.